docs/components/connectors/custom-built-connectors/connector-templates.md

```diff
@@ -442,6 +442,566 @@ a simple JSON configuration:
 </TabItem>
 </Tabs>
 
+## Inbound boundary event Connector templates
+
+You can, for example, allow the user to model and configure the following **HTTP webhook Connector** for boundary events by providing
+a simple JSON configuration:
+
+<Tabs groupId="connectorTemplateInbound" defaultValue="process" values={
+[
+{label: 'Process modeling view', value: 'process' },
+{label: 'Connector template configuration', value: 'json' }
+]
+}>
+
+<TabItem value='process'>
+
+![Webhook Inbound boundary Connector Example.png](img/custom-connector-template-inbound-boundary.png)
+
+</TabItem>
+
+<TabItem value='json'>
+
+```json
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Webhook Boundary Event Connector",
+  "id": "io.camunda.connectors.webhook.WebhookConnectorBoundary.v1",
```

Comment:
I think we should rather link to the marketplace then including the element template here.

Reply:
Is this still an open topic?

Reply:
Based on discussion with @mathias-vandaele and @sbuettner, I have added an admonition at the top of the page with a link to Marketplace. I have also backported this, though they may have additional changes based on casing for the elements in this PR.